test/utils.py

```diff
@@ -10,12 +11,14 @@
 import torch
 
 
-# For use with decoded data frames, or in other instances were we are confident that
-# reference and test tensors should be exactly equal. This is true for decoded data
-# frames from media because we expect our decoding to exactly match what a user can
-# do on the command line with ffmpeg.
+# For use with decoded data frames. On Linux, we expect exact, bit-for-bit equality. On
+# all other platforms, we allow a small tolerance.
 def assert_tensor_equal(*args, **kwargs):
-    torch.testing.assert_close(*args, **kwargs, atol=0, rtol=0)
+    if sys.platform == "linux":
+        absolute_tolerance = 0
+    else:
+        absolute_tolerance = 3
```

Comment:
Is there an issue here? Or are we just accepting that this will always remain here?

Maybe you can add the result of your investigation here so future code readers will know why we did this.

Reply:
The issue, I believe, is that FFmpeg does not guarantee bit-for-bit equality across architectures. I believe some version of this will always remain, unless we decide to partition our test tensors by system and architecture.

Reply:
I meant if we have a github issue here for reference. You should put this comment in the code as a comment because more people look at the code than the PR thread. That's what I meant -- add a comment here for future code readers (in the code itself, not the PR comment thread).

Reply:
Yup, I expanded on the comment that was already there.